src/dotnet/commands/dotnet-migrate/ProjectJsonParser.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Cli
+{
+    /// <summary>
+    /// Parses select data from a project.json without relying on ProjectModel.
+    /// Used to parse simple information.
+    /// </summary>
+    internal class ProjectJsonParser
+    {
+        public static string SdkPackageName => "Microsoft.DotNet.Core.Sdk";
```

Comment:
This was intended to be the source of truth for this property but then I moved this file. Removed into one location.